src/main/java/com/ams/restapi/authentication/SecurityConfig.java

```diff
@@ -34,7 +34,12 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 auth.requestMatchers("/admin/**").hasAuthority("ROLE_ADMIN");
                 auth.requestMatchers("/sections").hasAnyAuthority("ROLE_ADMIN", "ROLE_INSTRUCTOR");
                 auth.requestMatchers(HttpMethod.POST, "/attendance").permitAll();
-                auth.requestMatchers(HttpMethod.GET, "/attendance").hasAnyAuthority("ROLE_ADMIN", "ROLE_INSTRUCTOR");
+                auth.requestMatchers(HttpMethod.GET, "/attendance").permitAll(); // TODO: temporary to allow for testing, REMOVE LATER
```

Comment:
Remove these security disabling request matcher entries, this is instead dealt with by adding `@WithMockUser(roles="INSTRUCTOR")` annotations to test methods now.